backend/src/migrations/20240216181434_add_columns_to_user_details_table.ts

```diff
@@ -0,0 +1,39 @@
+import { type Knex } from 'knex';
+import { Gender } from 'shared';
```

Comment:
migrations should not rely on dependencies.

Reply:
Hmm, so I need to hardcode the values?